packages/pigeon/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## NEXT
```

Comment:
This will need to be updated to 22.7.1 as well as in the pubspec.yaml file and generator_tools.dart

Reply:
Done

Reply:
Now it is [complaining](https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8727396455078703345/+/u/Run_package_tests/CHANGELOG_and_version_validation/stdout) that version was not changed :)